src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.InlineRenameLocationSet.cs

```diff
@@ -42,9 +42,9 @@ public async Task<IInlineRenameReplacementInfo> GetReplacementsAsync(string repl
                 var conflicts = await _renameLocationSet.ResolveConflictsAsync(
                     _renameInfo.GetFinalSymbolName(replacementText), nonConflictSymbols: null, cancellationToken: cancellationToken).ConfigureAwait(false);
 
-                Contract.ThrowIfTrue(conflicts.ErrorMessage != null);
+                Contract.ThrowIfTrue(conflicts is FailedConflictResolution);
 
-                return new InlineRenameReplacementInfo(conflicts);
+                return new InlineRenameReplacementInfo((ConflictResolution)conflicts);
```

Comment:
how do you know this is safe?

Reply:
oh.  this is the C#/vb impl?

Reply:
Yes, it's the C#/vb impl.
The only case when this could fail is the rename symbol could not be found in source.
For inline rename I think it is guaranteed right? Becuase I saw in the original code it is supposing ErrorMessage is null